enterprise/frontend/src/embedding-sdk/components/public/StaticDashboard/CypressStaticDashboardWithCors.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import type { ComponentStory } from "@storybook/react";
+
+import { StaticDashboard } from "embedding-sdk";
+import { CommonSdkStoryCorsWrapper } from "embedding-sdk/test/CommonSdkCorsStoryWrapper";
+
+const DASHBOARD_ID = (window as any).DASHBOARD_ID || "1";
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: "EmbeddingSDK/CypressStaticDashboardWithCors",
+  component: StaticDashboard,
+  parameters: {
+    layout: "fullscreen",
+  },
+  decorators: [CommonSdkStoryCorsWrapper],
```

Comment:
Duplicate this file from `StaticDashboard.stories.tsx` but change this decorator to the done that doesn't sign JWT.